config/prow/plugins.yaml

```diff
@@ -247,6 +247,11 @@ blockades:
   exceptionregexps:
   - ^README\.md$
   explanation: "We do not accept changes directly against this repository, unless the change is to the `README.md` itself. Please see `CONTRIBUTING.md` for information on where and how to contribute instead."
+- repos:
+  - kubernetes/test-infra
+  blockregexps:
+  - ^prow/.+\.md$
+  explanation: "All Prow documentation (Markdown files) have moved to https://github.com/kubernetes-sigs/prow as part of https://github.com/kubernetes-sigs/prow/issues/4."
```

Comment:
I suspect we'll want some exceptions. In particular we're definitely still going to want a README for Prow as a whole, even if it only really points to the documentation site. We might want other READMEs too? WDYT?

We can tweak this later so we don't need to determine this now.

Reply:
Exception for main Prow README sounds fine. I'm not sure which other ones need exceptions, so I'd rather leave that up to future PRs as they come up.

Reply:
Added exception for `prow/README.md`.